presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcIntegrationSmokeTest.java

```diff
@@ -13,16 +13,124 @@
  */
 package io.prestosql.plugin.jdbc;
 
+import com.google.common.collect.ImmutableList;
+import io.airlift.tpch.TpchTable;
+import io.prestosql.Session;
 import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
+import io.prestosql.testing.sql.JdbcSqlExecutor;
+import io.prestosql.testing.sql.TestTable;
+import org.testng.annotations.Test;
 
-import static io.airlift.tpch.TpchTable.ORDERS;
+import java.util.Map;
+import java.util.Properties;
+
+import static io.prestosql.plugin.jdbc.BaseJdbcPropertiesProvider.UNSUPPORTED_TYPE_HANDLING;
 import static io.prestosql.plugin.jdbc.H2QueryRunner.createH2QueryRunner;
+import static io.prestosql.plugin.jdbc.UnsupportedTypeHandling.CONVERT_TO_VARCHAR;
+import static io.prestosql.plugin.jdbc.UnsupportedTypeHandling.IGNORE;
+import static java.lang.String.format;
 
 public class TestJdbcIntegrationSmokeTest
         extends AbstractTestIntegrationSmokeTest
 {
+    private final Map<String, String> properties;
+
     public TestJdbcIntegrationSmokeTest()
     {
-        super(() -> createH2QueryRunner(ORDERS));
+        this(TestingH2JdbcModule.createProperties());
+    }
+
+    public TestJdbcIntegrationSmokeTest(Map<String, String> properties)
```

Comment:
Please rebase on top of https://github.com/prestosql/presto/pull/2038 and remove constructor overloading here. (in fact, remove both ctors)